docs/dashboards/synchronize-components.md

```diff
@@ -81,3 +81,77 @@ Dashboards.board('container', {
 |[Highcharts](https://www.highcharts.com/docs/dashboards/highcharts-component)|yes|yes|yes
 |[DataGrid](https://www.highcharts.com/docs/dashboards/datagrid-component)|yes|yes|yes
 |[KPI](https://www.highcharts.com/docs/dashboards/kpi-component)|no|yes|no
+
+### Creating a Custom Synchronization
+By default, Dashboard supports three synchronization modes: `highlight`, `extremes` and `visibility`
+in its components. However, there may be situations where you require custom functionality tailored to your specific needs. In such cases, you can define your own custom synchronization.
+
+<iframe style="width: 100%; height: 470px; border: none;" src=https://www.highcharts.com/samples/embed/dashboards/component-options/custom-sync/ allow="fullscreen"></iframe>
+
+## How to start
+1. To enable custom synchronization, you must use the same connector between synchronized components, just as you would with the other types of synchronization modes (`highlight`, `extremes`, `visibility`).
+
+2. Define a unique event name within the sync object and reference the handler and emitter functions.
+
+```js
+sync: {
+    highlightMarker: {
+        handler: highlightMarkerHandler,
+        emitter: highlightMarkerEmitter
+    }
+}
+
+```
+
+Please note that the `highlightMarkerHandler` and `highlightMarkerEmitter` should be references to your own functions. Examples of these functions will be provided in the next step.
+
+3. Handler
+The handler is responsible for receiving input from the connector and taking appropriate actions. The handler can be divided into three main parts.
+
+First, the section where you register your syncs:
+
+```js
+if (board) {
+    registerCursorListeners();
+
+    this.on('setConnector', () => unregisterCursorListeners());
+    this.on('afterSetConnector', () => registerCursorListeners());
+}
+```
+
+Second, the definition of listeners:
+
+```js
+const { dataCursor: cursor } = board;
+cursor.addListener(table.id, 'point.mouseOver', handleCursor);
+cursor.addListener(table.id, 'point.mouseOut', handleCursorOut);
+```
+
+Listeners define the actions to be taken when a trigger event occurs.
+Don't forget to unregister these events when necessary:
+
+```js
+board.dataCursor.removeListener(table.id, 'point.mouseOver', handleCursor);
+board.dataCursor.removeListener(table.id, 'point.mouseOut', handleCursorOut);
+```
+
+The final step is to define your custom actions, which should be triggered.
```

Comment:
Suggestion: "The final step is to define your custom `handleCursor` and `handleCursorOut` functions."

Maybe a condensed example as well?

Reply:
I'm not convinced about the `"The final step is to define your custom handleCursor and handleCursorOut functions."`, because the names can be any.

Reply:
That's true. It's mostly the "which should be triggered" part that is triggering me. 

Maybe it could be "which are triggered by the event listeners above"?

Also maybe "custom actions" -> "callback functions"?